pkg/webhook/admission/response.go

```diff
@@ -54,7 +56,23 @@ func ValidationResponse(allowed bool, reason string) types.Response {
 	return resp
 }
 
-// PatchResponse returns a new response with json patch.
+// PatchResponseFromRaw takes 2 byte arrays and returns a new response with json patch.
+func PatchResponseFromRaw(original, current []byte) types.Response {
```

Comment:
I prefer this over the other PR which takes three params `original, current, originalRawBytes` which could get confusing for the end user and also not help user avoid the round-tripping issue.